api/v2/views/access_token.py

```diff
@@ -22,7 +22,8 @@ def get_queryset(self):
         return AccessToken.objects.filter(token__user=self.request.user)
 
     def create(self, request):
-        AccessTokenSerializer().validate(request.data)
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
```

Comment:
:+1: May seem a tad overkill for so simple an api, but this future proofs as more fields get added

Reply:
Better to do it right the first time for sure. I appreciate all the feedback, I am learning a lot about APIs!